TACC/Core-Portal
# Add ARIA label to Data Files checkboxes for accessibility

## Overview: ##
If DataFilesListing checkboxes are not using native HTML, then they should be given the same accessibility features. This can be done by providing 'aria-label' or 'aria-labelledby' attributes.

## Related Jira tickets: ##

* [FP-1385](https://jira.tacc.utexas.edu/browse/FP-1385)

## Summary of Changes: ##
- Changed `role` from "button" to checkbox"
- Added an `id` and `aria-label` for the checkboxes

## Testing Steps: ##
1. (Optional) Turn on VoiceOver (CMD+F5) or any other accessibility reader and confirm that the labels work correctly.

## UI Photos:
https://user-images.githubusercontent.com/51714485/146410999-61a13c61-7e2a-4e8c-a0ae-3c8355fb682e.mov

## Notes: ##
